crates/uv/src/commands/project/format.rs

```diff
@@ -40,8 +40,19 @@ pub(crate) async fn format(
 
     let workspace_cache = WorkspaceCache::default();
     let project =
-        VirtualProject::discover(project_dir, &DiscoveryOptions::default(), &workspace_cache)
-            .await?;
+        VirtualProject::discover(project_dir, &DiscoveryOptions::default(), &workspace_cache).await;
+    let working_dir = match project {
```

Comment:
If we don't re-use the `project` binding I might just do `match VirtualProject::discover(...)` directly.